src/renderer/html_handlebars/hbs_renderer.rs

```diff
@@ -121,12 +121,10 @@ impl HtmlHandlebars {
     }
 
     fn post_process(&self, rendered: String) -> String {
-        let rendered = build_header_links(rendered, "print.html");
-        let rendered = fix_anchor_links(rendered, "print.html");
-        let rendered = fix_code_blocks(rendered);
-        let rendered = add_playpen_pre(rendered);
-
-        rendered
+        let rendered = build_header_links(&rendered, "print.html");
+        let rendered = fix_anchor_links(&rendered, "print.html");
+        let rendered = fix_code_blocks(&rendered);
+        add_playpen_pre(&rendered)
```

Comment:
I understand why Clippy suggests this, but I am of the opinion that the previous code is actually somewhat clearer and more coherent in this case. No?